### DISCUSSION TITLE
photon and nominatim import athene

### DISCUSSION THREAD

**Discussion by matro91** [komoot/photon#573] [Q&A]
Hi installed photon and nominatim. when i did curl http://127.0.0.1:2322/api?q=Berlin everythink is fine, but if i try
curl http://127.0.0.1:2322/api?q=Athenes or curl http://127.0.0.1:2322/api?q=Athènes it doesn't returne Athene in greece (as i made my test on europe only it juste return me "les vignes d'Athènes" in paris) bur if i try curl http://127.0.0.1:2322/api?q=Αθήνα i have all the résults even the on i had with curl http://127.0.0.1:2322/api?q=Athenes .
thanks for your help
Mathilde

**Comment by lonvia** [comment]
Photon is rather sensitive to your chosen language. If you request results in French, it will work: https://photon.komoot.io/api/?debug=1&q=Ath%C3%A8nes&lang=fr

If you search with the default language (that happens when using curl without a lang parameter), it prefers results that have the word exactly in the name: https://photon.komoot.io/api/?debug=1&q=Ath%C3%A8nes
